src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {new Truck(), new Bulldozer(), new Excavator()};
+        for (Machine element : machines) {
```

Comment:
```suggestion
        for (Machine machine : machines) {
```

Reply:
Thank you, I'll take your comment into consideration.